src/pages/open-source-for-devs.md

```diff
@@ -0,0 +1,32 @@
+Hey there, developer!
+
+If you’ve heard about open source but are unsure what it is or where to start, you’ve come to the right place. We’ll explain what open source means, the value of open source, and how it works.
+
+_Note: Open source is not just for developers. We welcome everyone to get involved! That’s why we’re creating explainers for folks from different professional backgrounds. Help us explain the value of open source to non-developers, companies, finance departments, project managers, and technical leaders._
+
+##What is Open Source?
+Let’s talk about what open source means. Originally [coined by Christine Peterson](https://opensource.com/article/18/2/coining-term-open-source-software), the term open source describes software that anyone can share or modify because the source code, or the code used to create the software, is publicly accessible. The opposite of such software is proprietary or “closed source” software. The code for this type of software is only legally accessible by the organization or individual who owns it.
```

Comment:
[proofreading]: Hi @blackgirlbytes! I made some suggestions on how to make the definition more concise:
1. I think it'd be helpful to just start explaining the definition instead of announcing it because the title already implies that this section is defining open source.
2. Consider making the part where you compared open source to closed source software a bit shorter so that company leaders can make those distinctions quickly.

Other than, you're off to a great start! :)  
```suggestion
Originally [coined by Christine Peterson](https://opensource.com/article/18/2/coining-term-open-source-software), the term, open source describes software containing code that anyone can share or modify.  Unlike software that has proprietary code, aka, “closed source”, the code used to create open source software is publicly accessible. Also, it is legally owned by the organization or individual who originally created the software.
```